Expand AI guidance (#1882)
CodeYourFuture/curriculum: org-cyf-guides/content/ai-usage/index.md

description = 'How we use AI at CYF to prioritise learning'
+++

AI tools are changing the way the tech industry are approaching software engineering. We know that at CYF we are prioritising people learning excellent technical skills so they can get good jobs in tech.

At CYF we have found that AI tools are changing the way people learn in a range of ways. Observed uses includes:

- Making concepts easier to understand, explaining back ideas.
- generating code to solve problems we're giving trainees to demonstrate concepts.

> - Using AI to complete large portions of projects

Reviewer: ```suggestion
- Using AI to complete large portions of projects.
```

- Problem solving a task they can then code based off of.
- asking difficult questions

> - generating written work, emails etc.

Reviewer: ```suggestion
- Asking difficult questions.
- Generating written work, emails etc.
```

> To do enable people to learn necessary skills to land good jobs in tech we have to provide guardrails and guidance around what acceptable AI use is across our courses.

Reviewer: ```suggestion
To enable people to learn necessary skills to land good jobs in tech we have to provide guardrails and guidance around what acceptable AI use is across our courses.
```

## AI Use Guidelines

> There is a full set of AI guidelines [here](/ai-use-guidelines).

Reviewer: ```suggestion
Read [our full set of AI guidelines](/ai-use-guidelines).
```

These guidelines should outline to everyone how to spot useful vs unhelpful AI use when it comes to our learning goals. It also outlines how acceptable AI use changes as we progress through the courses, from ITD all the way to Launch. The reason these guidelines change across the courses is because we have found we can use AI more accurately when we have the skills to question, re-prompt, or even correct the output of generative AI tools.

## Volunteer Cheatsheet

Are you a volunteer who needs a quick reference to help work with students using AI?

> Find a shorter "Cheat Sheet" of how to navigate AI usage with trainees across the courses [here](/ai-use-cheat-sheet).

Reviewer: ```suggestion
You can also find a shorter ["Cheat Sheet" of how to navigate AI usage with trainees across the courses](/ai-use-cheat-sheet).
```
